corehq/apps/api/odata/serializers.py

```diff
@@ -175,3 +176,55 @@ def serialize_cases_using_config(cases, config):
             }
             for case_data in cases
         ]
+
+
+class ODataFormSerializer(Serializer):
```

Comment:
This is nearly identical to `ODataCaseSerializer` except for the id field and the view it redirects to.  Do you have plans to deduplicat those?  Do you expect further variation to arise?

Reply:
I'm not sure if further variation will arise.  I'd probably recommend considering deduplication after the beta has started.